### ISSUE TITLE
Loading a font with it's extension in upper case throw an error -> MissingAssetLoader

### ISSUE THREAD

**Issue by Archina** [bevyengine/bevy#1425]
**Bevy version**

0.4.0

**Operating system & version**

Ubuntu 20.04

**What you did**

```rust
pub fn display_hello_world(
    commands: &mut Commands,
    asset_server: Res<AssetServer>
) {
    commands.spawn(TextBundle{
        text: Text{
            value: format!("Hello World!"),
            font: asset_server.load("fonts/256BYTES.TTF"),
            ..Default::default()
        },
        ..Default::default()
    });
}
```

**What you expected to happen**

The text to be loaded and the font to be displayed.

**What actually happened**

Bevy throws a panic within an IO Task Pool:

```
thread 'IO Task Pool (1)' panicked at 'called `Result::unwrap()` on an `Err` value: MissingAssetLoader(Some("TTF"))', /home/archina/.cargo/registry/src/github.com-1ecc6299db9ec823/bevy_asset-0.4.0/src/asset_server.rs:298:60
```

**Additional information**

Renaming the font to `<font_name>.ttf` fixes the error and load the font correctly.

**Comment by digital7-code** [comment]
In font_loader.rs try to change this line:
&["ttf", "otf"]
with this one:
&["ttf", "TTF", "otf", "OTF"]
and see if it still panics

**Comment by Archina** [comment]
I think a better way would be to use use `to_lower_case` before comparing as `TTf` and `tTF` are still not covered otherwise.

**Comment by alice-i-cecile** [comment]
Fixed by #1427.
